lib/beacon/runtime_css.ex

```diff
@@ -21,10 +21,17 @@ defmodule Beacon.RuntimeCSS do
   @doc false
   def load(site) do
     {:ok, css} = compile(site)
-    compressed = :zlib.gzip(css)
-    hash = Base.encode16(:crypto.hash(:md5, css), case: :lower)
-    true = :ets.insert(:beacon_assets, {{site, :css}, {hash, css, compressed}})
-    :ok
+
+    case :brotli.encode(css) do
+      {:ok, compressed} ->
+        hash = Base.encode16(:crypto.hash(:md5, css), case: :lower)
+        true = :ets.insert(:beacon_assets, {{site, :css}, {hash, css, compressed}})
+        :ok
+
+      error ->
```

Comment:
Not sure if we care about the error here, but included it just in case. Happy to remove it.

Reply:
That's fine. Raising here means that a site wouldn't load but on the other hand starting a site without styles and JS isn't a good option either.
The alternative is fall backing to gzip or uncompressed but it's not worth it unless we start having issues.